apps/full-site-editing/full-site-editing-plugin/donations/blocks/donations/edit.js

```diff
@@ -72,8 +72,15 @@ const Edit = ( { attributes, setAttributes } ) => {
 	};
 
 	useEffect( () => {
-		const updateData = () => fetchStatus( 'donation' ).then( mapStatusToState, apiError );
-		updateData();
+		const getStatus = async () => {
```

Comment:
This can be reverted. I changed this to debug easier a failure on my dev, but forgot to leave as it was before.